core/primitives/digest.hpp

```diff
@@ -31,6 +29,34 @@ namespace kagome::primitives {
   /// trie creation.
   struct ChangesTrieRoot : public common::Hash256 {};
 
+  // TODO (kamilsa): workaround unless we bump gtest version to 1.8.1+
+  // after gtest update use `data` type directly
+  struct ChangesTrieSignal {
+    boost::variant<boost::optional<storage::changes_trie::ChangesTrieConfig>> data;
+
+    bool operator==(const ChangesTrieSignal &rhs) const {
+      return data == rhs.data;
+    }
+
+    bool operator!=(const ChangesTrieSignal &rhs) const {
+      return !operator==(rhs);
+    }
+  };
+
+  template <class Stream,
+            typename = std::enable_if_t<Stream::is_encoder_stream>>
+  Stream &operator<<(Stream &s, const ChangesTrieSignal &sig) {
+    return s << sig.data;
+  }
+
+  template <class Stream,
+            typename = std::enable_if_t<Stream::is_decoder_stream>>
+  Stream &operator>>(Stream &s, ChangesTrieSignal &sig) {
+    return s >> sig.data;
+  }
+
+  struct Other : public common::Buffer {};
```

Comment:
Could we just use `using Other = common::Buffer`?

Reply:
As I see the `Other` struct is used as `DigestItem` value. Thus why do we have to inherit `Buffer` then?
Seems we could use just `struct Other {};`

Reply:
Yeah, but Other should contain array of bytes. I wanted to have a defined type like in substrate.